__EXAMPLES/interfaces/ring_section.py

```diff
@@ -10,96 +18,127 @@
 # BLonD Common import
 sys.path.append('./../../../')
 from blond_common.interfaces.input_parameters.ring_section import \
-    Section, machine_program
+    RingSection, machine_program
 
 
-# To declare a Section with simple input
-section_length = 300
+# To declare a RingSection with simple input
+length = 300  # m
 alpha_0 = 1e-3
-momentum = 26e9
+momentum = 26e9  # eV/c
 
-section = Section(section_length, alpha_0, momentum)
+section = RingSection(length, alpha_0, momentum)
 
-print(section.section_length,
+print(section.length,
       section.synchronous_data,
       section.alpha_0)
 
-# To declare a Section using other type of synchronous data
+# To declare a RingSection using other type of synchronous data
 # Note that to pass bending_field, bending_radius should be
 # passed as well
-section_length = 300
+length = 300  # m
 alpha_0 = 1e-3
-energy = 26e9
+energy = 26e9  # eV
 
-section = Section(section_length, alpha_0, energy=energy)
+section = RingSection(length, alpha_0, energy=energy)
 
-print(section.section_length,
+print(section.length,
       section.synchronous_data,
       section.alpha_0)
 
-# Turn-by-turn input
-section_length = 300
+# To declare a RingSection with turn-by-turn program as input
+length = 300  # m
 alpha_0 = [1e-3, 1e-3, 1e-3]
-momentum = [26e9, 27e9, 28e9]
+momentum = [26e9, 27e9, 28e9]  # eV
```

Comment:
eV/c

Reply:
Corrected everywhere (except the places where I forgot)